frontend/web/components/pages/AuditLogPage.js

```diff
@@ -1,172 +0,0 @@
-import React, { Component } from 'react';
```

Comment:
I think this file can be completely removed? Seems like we've just truncated it?